sdks/java/testing/nexmark/src/main/java/org/apache/beam/sdk/nexmark/Main.java

```diff
@@ -190,7 +189,7 @@ static void savePerfsToBigQuery(
               .put("numResults", entry.getValue().numResults)
               .build();
 
-      bigQueryClient.insertRow(record, tableName);
+      bigQueryClient.insertRow(record, schema, tableName);
```

Comment:
Should this test be changed to use the ResultPublisher? I see that the schemas are a tad different? `runtimeSec` vs `runtime`, etc?